### PR TITLE
[624] BARN//PRODUCTION config setup

### LINKED ISSUE #624: Split config environemnts (prod/barn)
**Description**

Split `configuration` branch settings in another dimension: environment

Right now, it's split by network, but any changes made will apply to both PROD and BARN.

We should make them independent to allow us to test new features without impacting running services on production

### PR DESCRIPTION
**Pull request by W3stside**
# Summary

Closes #624 

FS was:
```
config ----
    announcements ----
        announcement-1.json
        .. etc ..
    
    strategies ----
        strategy-1.json
        .. etc ..
```
is now split at the root:
```
config ----
    barn ----
        announcements ----
            announcement-1.json
            .. etc ..
    
        strategies ----
            strategy-1.json
            .. etc ..
    production ----
        ... etc ...
```  

## Note
can ignore the build breaking because of sentry. This branch is not updated against the main branches thus is missing a ton of essential code. Only relevant changes are in the `config` folder

### CODE REVIEW

File: config/production/announcements/phishing-template.md [hunk 1]
```diff
@@ -0,0 +1 @@
+**IMPORTANT**: If you are getting a message about a CowSwap token sale - **THIS IS A SCAM!** For more info click [here](https://discord.com/channels/869166959739170836/869948218388119584/911951973547393126)
```
Comment (alfetopito) [inline config/production/announcements/phishing-template.md:1]: Remove?

File: config/production/announcements/announcement-contracts.md [hunk 2]
```diff
@@ -0,0 +1 @@
+**IMPORTANT**: Contracts have been recently updated. Read more [here](https://docs.gnosis.io/protocol/)!
```
Comment (alfetopito) [inline config/production/announcements/announcement-contracts.md:1]: Remove?

**Review by alfetopito — APPROVED** [review]

**Comment by alfetopito** [comment]
Also, we should either:
1. Hold off merge until https://github.com/cowprotocol/cowswap/pull/641 is deployed to prod
2. Keep current config and merge the barn/prod split. When https://github.com/cowprotocol/cowswap/pull/641 reaches prod, remove the current flat structure

**Comment by W3stside** [comment]
after discussion with @alfetopito he correctly pointed out that after the FS change there would be a period in time in which one of the builds wouldn't work. So i added the original prod files back into root which we can remove later

**Review by alfetopito — APPROVED** [review]
Ready to merge

**Pull request merged**